js/calendar.js

```diff
@@ -1089,8 +1090,8 @@ if(!String.prototype.formatNum) {
 		var self = this;
 		var activecell = 0;
 		var downbox = $(document.createElement('div')).attr('id', 'cal-day-tick').html('<i class="icon-chevron-down glyphicon glyphicon-chevron-down"></i>');
-
-		$('.cal-month-day, .cal-year-box .span3')
+		var calendarSelector = self.context.selector;
+		$(calendarSelector + ' .cal-month-day, .cal-year-box .span3')
```

Comment:
Would it be better to use context like this

    $('.cal-month-day, .cal-year-box .span3', self.context)

Then it will look inside calendar. And it is cleaner. Also your way it should be

    $(calendarSelector + ' .cal-month-day, ' + calendarSelector + '.cal-year-box .span3')